src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -774,4 +774,44 @@ public ref partial struct SpanRuneEnumerator
         public System.Text.SpanRuneEnumerator GetEnumerator() { throw null; }
         public bool MoveNext() { throw null; }
     }
+    public readonly partial struct RunePosition : System.IEquatable<System.Text.RunePosition>
+    {
```

Comment:
Review feedback mentioned: `We should have a constructor that sets all four fields`. I think this constructor is missing from this implementation.